src/ansys/acp/core/_tree_objects/model.py

```diff
@@ -412,6 +424,75 @@ def export_materials(self, path: _PATH) -> None:
                 )
             )
 
+    @supported_since("25.1")
+    def export_modeling_ply_geometries(
+        self,
+        path: _PATH,
+        *,
+        format: PlyGeometryExportFormat = PlyGeometryExportFormat.STEP,
+        offset_type: OffsetType = OffsetType.MIDDLE_OFFSET,
+        include_surface: bool = True,
+        include_boundary: bool = True,
+        include_first_material_direction: bool = True,
+        include_second_material_direction: bool = True,
+        arrow_length: float = 1.0,
```

Comment:
Not sure if that is a reasonable default. What about something like `model.average_element_size`?

Reply:
Yeah, square root of `average_element_size` is consistent with the ACP GUI.